README.md

```diff
@@ -153,17 +153,20 @@ The `uswds` module is now installed as a dependency. You can use the un-compiled
 ```
 node_modules/uswds/
 ├── dist/
+│   ├── CONTRIBUTING.md
+│   ├── LICENSE.md
+│   ├── README.md
```

Comment:
I'd probably remove the `md` files docs from this example code as their not super vital and we're not showing everything in the folder in this example anyway. Other than that, LGTM.